### DISCUSSION TITLE
Non-Inventory Product

### DISCUSSION THREAD

**Discussion by mraf** [givanz/Vvveb#114] [Ideas]
Is there a plan maybe to have option Don't track inventory for products? 

Will be good for digital product for example or main product where only parts ( options ) are tracked,

**Comment by givanz** [comment]
There is a `Subtract Stock` option in the inventory tab on the product edit page to disable stock subtraction per product https://demo.vvveb.com/admin/index.php?module=product/product&product_id=19&type=product

Is this option fit to disable inventory tracking for your use case or you need other options?

**Reply by mraf** [reply]
yes it's great! I missed this setting somehow :)
